### DISCUSSION TITLE
LLaVA install 出了点问题

### DISCUSSION THREAD

**Discussion by CaoYFan** [haotian-liu/LLaVA#135] [Q&A] [answered]
![image](https://github.com/haotian-liu/LLaVA/assets/73150026/82ff2914-a250-4793-87fa-dd6a5f114343)
![image](https://github.com/haotian-liu/LLaVA/assets/73150026/21ed360c-c269-4268-b6ce-7d400c737eab)
亲爱的开发者们，在下载这个包的时候出现了问题，可以帮忙看看怎么解决吗？拜托了

**Comment by haotian-liu** [comment, marked as answer]
Hi, it takes some time to build flash-attn.  Please make sure that you have installed ninja first and wait for 10-15 mins.  You may use top to see if the compilation is running.  If you see an error message, please paste them here, and wrap with ``` code block.  Thanks.

**Reply by CaoYFan** [reply]
谢谢老哥，我再去试一下。
